packages/solr.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# The following script is pulling the binary files from the
+# recommended mirror site. Visit http://www.apache.org/dyn/closer.lua/lucene/solr/
+# for a full list of available mirrors.
+#
+# Remember to set the necessary version of Java. See
+# https://codeship.com/documentation/languages/java-and-jvm-based-languages/
+# for more information.
+#
+# To run this exact configuration, add the following to your builds:
+curl -sSL https://raw.githubusercontent.com/codeship/scripts/master/packages/solr.sh | bash -s
```

Comment:
That should be a comment as well, otherwise, whoever runs the `curl` command will run this `curl` command as well, will run this `curl`, ... (looping)